### DISCUSSION TITLE
Black Pop-Up come When Building Using Cordova in iOS

### DISCUSSION THREAD

**Discussion by Nidhishar** [quasarframework/quasar#15001] [CLI - Capacitor / Cordova modes]
I am opening this ticket as I am getting an issue while I am building an IOS dev build of an app that includes a video player to a simulator. After the build is completed, playing the player results in a black pop-up that covers the page. It's a SPA

If I click the black box, I can see a few controls for pausing or forwarding/reversing, plus a close box (X). Clicking the X makes the pop-up go away.

To Reproduce

**Create a Quasar app using a player**
Run: quasar dev -m cordova -T ios
Play the player in the app

**Expected behavior**
I expect the player to play without popping up a black box that blocks the rest of the page.

![92790384-bd417f00-f379-11ea-8b17-7a571b512e4e](https://user-images.githubusercontent.com/33097032/204551607-b355144d-8fd6-45d1-bc85-806dec6a4461.png)

The same app is running perfectly on the Android build. 
Please share your thoughts and some solutions to solve this problem. 

Thanks in advance.

**Comment by pdanpdan** [comment]
This is related to how video tag works in iOS. Place check mdn docs for video tag - related to play inline.
